deploy/templates/notificationtemplates/appstudio/userdeactivating/notification.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>
+        Notice: Your account will be deactivated soon.
+    </title>
+    <style>
+        a:hover {
+            text-decoration: underline !important;
+        }
+        p {
+            text-align: left;
+            margin: 30px 0;
+        }
+    </style>
+</head>
+
+<body
+        style="
+       padding: 10px;
+       padding: 0;
+       background-color: #f9f9f9;
+       font-family: 'Open Sans', sans-serif;
+       font-size: 15px;
+       font-weight: lighter;
+       line-height: 1.2;"
+>
+<div
+        style="
+       min-height: 300px;
+       max-width: 750px;
+       margin: 0 auto;
+       padding: 20px;
+       border: 1px solid #d7d7d7;
+       border-radius: 4px;
+       background-color: #fff;
+       box-shadow: 0 2px 4px #d7d7d7;"
+>
+
+    <p>
+        You are receiving this email because your email account {{.UserEmail}} was provisioned.
+    </p>
+
+    <p>
+        Your account will expire in 3 days.  We recommend you save your work as all data will be
+        deleted upon expiry.  After deactivation, you can sign up again at any time for new access at {{.RegistrationURL}}.
```

Comment:
out of curiosity - is it the plan to set the registration URL in ToolchainConfig to HAC? https://github.com/codeready-toolchain/sandbox-sre/blob/master/resources/stone-prd-host1.wdlc.p1.openshiftapps.com/configure/host/sandbox/toolchainconfig.yaml#L49

Reply:
Right. We need to set it to HAC URL instead. Similar to what we did for sandbox prod: https://github.com/codeready-toolchain/sandbox-sre/blob/master/resources/sandbox.x8i5.p1.openshiftapps.com/configure/host/sandbox/toolchainconfig.yaml#L50
In sandbox prod we set it to RHD.
In stonesoup we need to set it to HAC.

@ranakan19 let's add this `{{.RegistrationURL}}.` to the welcome email too! And we will update it in the configuration for the environments in sandbox-sre.

Reply:
https://github.com/codeready-toolchain/sandbox-sre/pull/956

Reply:
And see my suggestion for the welcome email below.